### ISSUE TITLE
Cannot link to documentation title anchors

### ISSUE THREAD

**Issue by kumar303** [mozilla/extension-workshop#497]
### Describe the problem and steps to reproduce it:

* Go to https://extensionworkshop.com/documentation/develop/getting-started-with-web-ext/
* Scroll down to *Using web-ext from a script*

### What happened?

You cannot link to this section of the documentation.

### What did you expect to happen?

MDN had an anchor icon which let you link to titles.

**Comment by caitmuenster** [comment]
Thanks @kumar303. Would it be helpful if we anchor linked all of the h3s on this page?

**Comment by kumar303** [comment]
Yeah, any header (h2, h3, h4, ...) should be linkable. I guess linking h1s wouldn't be so helpful but maybe.

**Comment by caitmuenster** [comment]
That should be doable! The h2s should already have anchors -- I'll add the h3 anchors for this week's push.

**Comment by muffinresearch** [comment]
Anchors weirdly seem to be on the sections. It would be good to move them to the headings, then we could more easily add some script to provide a link to the heading, which is a pretty established pattern.

That said looking at MDN I don't see any special behaviour for this either.

<img width="1280" alt="Getting_started_with_web-ext___Extension_Workshop" src="https://user-images.githubusercontent.com/1514/66563207-31b1e380-eb55-11e9-852c-9c58abde4537.png">

**Comment by kumar303** [comment]
Also, as a bonus feature it would be fantastic if old links redirected with the anchors in tact 🍰 🎠 

Example: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/web-ext_command_reference#web-ext_run 

^ This used to link directly to the `run` command. Using *find on page* to locate the *web-ext run* section is quite difficult since it's mentioned so many times.

**Comment by muffinresearch** [comment]
Also note, that we seem to have turned off auto-id generation here: https://github.com/mozilla/extension-workshop/blob/6c83d45da0952a4e8e843b00c72409b4d03019e5/_config.yml#L38
